Userland/Libraries/LibConfig/Client.cpp

```diff
@@ -30,6 +30,16 @@ void Client::monitor_domain(String const& domain)
     async_monitor_domain(domain);
 }
 
+Vector<String> Client::list_keys(AK::StringView domain, AK::StringView group)
```

Comment:
Please don't qualify classes from `AK`, i.e.
```suggestion
Vector<String> Client::list_keys(StringView domain, StringView group)
```

(similar case below)

Reply:
whoops, auto completion put that in there :^). Amended commit and fixed it.